packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1490,6 +1493,10 @@ export const components = ({
             marginBlockEnd: 0,
           },
           textAlign: "start",
+
+          ".field-hint-link-component": {
```

Comment:
I left a question earlier about whether we can use the component name here instead of a class name

Reply:
I just wrapped it in a `Box` and styled it there. I agree this class shouldn't be here. Thanks for the heads up